pineappl_py/pineappl/grid.py

```diff
@@ -193,38 +191,38 @@ def convolve_with_one(
         lumi_mask=np.array([], dtype=bool),
         xi=((1.0, 1.0),),
     ):
-        r"""Convolute grid with pdf.
+        r"""Convolve with a single distribution.
 
         Parameters
         ----------
-            pdg_id : int
-                PDG Monte Carlo ID of the hadronic particle `xfx` is the PDF for
-            xfx : callable
-                lhapdf like callable with arguments `pid, x, Q2` returning x*pdf for :math:`x`-grid
-            alphas : callable
-                lhapdf like callable with arguments `Q2` returning :math:`\alpha_s`
-            order_mask : sequence(bool)
-                Mask for selecting specific orders. The value `True` means the corresponding order
-                is included. An empty list corresponds to all orders being enabled.
-            bin_indices : sequence(int)
-                A list with the indices of the corresponding bins that should be calculated. An
-                empty list means that all orders should be calculated.
-            lumi_mask : sequence(bool)
-                Mask for selecting specific luminosity channels. The value `True` means the
-                corresponding channel is included. An empty list corresponds to all channels being
-                enabled.
-            xi : list((float, float))
-                A list with the scale variation factors that should be used to calculate
-                scale-varied results. The first entry of a tuple corresponds to the variation of
-                the renormalization scale, the second entry to the variation of the factorization
-                scale. If only results for the central scale are need the list should contain
-                `(1.0, 1.0)`.
+        pdg_id : int
+            PDG Monte Carlo ID of the hadronic particle
+        xfx : callable
+            lhapdf like callable with arguments `pid, x, Q2` returning x*pdf for :math:`x`-grid
+        alphas : callable
+            lhapdf like callable with arguments `Q2` returning :math:`\alpha_s`
+        order_mask : sequence(bool)
+            Mask for selecting specific orders. The value `True` means the corresponding order
+            is included. An empty list corresponds to all orders being enabled.
+        bin_indices : sequence(int)
+            A list with the indices of the corresponding bins that should be calculated. An
+            empty list means that all orders should be calculated.
+        lumi_mask : sequence(bool)
```

Comment:
Here you should write `channel_mask` or something similar. Also replace luminosity with channels in the text.

Reply:
0d2866c2